### ISSUE TITLE
Tabs class

### ISSUE THREAD

**Issue by istratievladut** [react-toolbox/react-toolbox#1510]
When you select a tab, it receives a class "undefined".

![screen shot 2017-05-30 at 11 06 53](https://cloud.githubusercontent.com/assets/25583466/26574005/7499dd90-4528-11e7-89e7-4b83df98ab85.png)

**Comment by tobiasandersen** [comment]
It could be that you haven't passed a value to the `activeClassName` prop. If you don't want to define a real class name, just pass an empty string.

**Comment by istratievladut** [comment]
Yeah, that was the problem. But it shouldn't check if it receive an activeClassName as prop?

**Comment by tobiasandersen** [comment]
Yeah I guess that would be preferable...

**Comment by KhushilMistry** [comment]
Hey, I want to work on this bug.
Can't we add activeClassName into default props as empty string ?

**Comment by rubenmoya** [comment]
The problem is in [Tab.js#L64](https://github.com/react-toolbox/react-toolbox/blob/dev/components/tabs/Tab.js#L64), we're only checking for `active` in the conditional, you can check how [classnames](https://github.com/JedWatson/classnames#usage) works.

I think we should be fine by adding `activeClassName` to that check something like `active && activeClassName` would do the trick :)

Feel free to work on it @KhushilMistry!
